.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
-This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
+Hello & Welcome to our repo,
 
-### PR information
-- [ ] The title of the PR is clear and informative.
-- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
-- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
-- [ ] If applicable, the PR references the bug/issue that it fixes.
-- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
+You Can find a link to our review process [here](https://github.com/Azure/adx-documentation-pr/wiki/swagger-management). if you can't access this link, register your github your account to [Azure Github](https://aka.ms/azuregithub).
```

Comment:
minor: "Can" should be all lower case in "You can find a link..."
minor: start "If you can't..." with capital letter.

Reply:
Agreed